catalog/ddms/catalog-ddms-transformer/src/main/java/org/codice/alliance/ddms/v2/Ddms20InputTransformer.java

```diff
@@ -0,0 +1,762 @@
+
+    // Associated attributes
+    metacard.setAttribute(getDerivedAssociations(ddms));
+    metacard.setAttribute(getRelatedAssociations(ddms));
+    metacard.setAttribute(getExternalAssocations(ddms));
+
+    // Datetime attributes
+    metacard.setAttribute(getDatetimeNames(ddms));
+    metacard.setAttribute(getDatetimeStart(ddms));
+    metacard.setAttribute(getDatetimeEnd(ddms));
+
+    // Contact attributes
+    metacard.setAttribute(getCreatorNames(ddms));
+    metacard.setAttribute(getCreatorEmails(ddms));
+    metacard.setAttribute(getCreatorPhones(ddms));
+    metacard.setAttribute(getPublisherNames(ddms));
+    metacard.setAttribute(getPublisherEmails(ddms));
+    metacard.setAttribute(getPublisherPhones(ddms));
+    metacard.setAttribute(getContributorNames(ddms));
+    metacard.setAttribute(getContributorEmails(ddms));
+    metacard.setAttribute(getContributorPhones(ddms));
+    metacard.setAttribute(getPointOfContactNames(ddms));
+    metacard.setAttribute(getPointOfContactEmails(ddms));
+    metacard.setAttribute(getPointOfContactPhones(ddms));
+
+    // Location attributes
+    metacard.setAttribute(getCountryCodes(ddms));
+    metacard.setAttribute(getCrsNames(ddms));
+    // TODO: location.altitude-meters is a content extension
+
+    // Media attributes
+    metacard.setAttribute(getMediaFormat(ddms));
+    metacard.setAttribute(getMediaType(ddms));
+
+    // Topic attributes
+    metacard.setAttribute(getTopicCategories(ddms));
+    metacard.setAttribute(getTopicKeywords(ddms));
+    metacard.setAttribute(getTopicVocabulary(ddms));
+
+    // @TODO: isr.frequency-hertz is a content extension
+
+    // Security attributes
+    SecurityAttributes securityAttributes = ddms.getSecurity().getSecurityAttributes();
+    metacard.setAttribute(getClassification(securityAttributes));
+    metacard.setAttribute(getReleasability(securityAttributes));
+    metacard.setAttribute(getOwnerProducers(securityAttributes));
+    metacard.setAttribute(getDisseminationControls(securityAttributes));
+    metacard.setAttribute(getSciControls(securityAttributes));
+    metacard.setAttribute(getNonIcMarkings(securityAttributes));
+
+    // Restriction attributes
+    metacard.setAttribute(getCopyright(ddms));
+    metacard.setAttribute(getIntellectualProperty(ddms));
+    metacard.setAttribute(getPrivacyAct(ddms));
+
+    // Ddms attributes
+    metacard.setAttribute(getInfoCutOff(ddms));
+    metacard.setAttribute(getPosted(ddms));
+    metacard.setAttribute(getGeographicNames(ddms));
+    metacard.setAttribute(getGeographicRegions(ddms));
+    metacard.setAttribute(getBeNumbers(ddms));
+    metacard.setAttribute(getSubtitles(ddms));
+    // TODO: ext.ddms.icid-payload
+    // TODO: ext.ddms.security portion markings
+    // Portion-Markings: Title, Subtitle, description, contacts, related resources, security
+    // TODO: ext.ddms.related-resource not used?
+    // Todo: Not sure what ext.ddms.topic-type is for
```

Comment:
Should these TODO's reference a future ticket or be removed?

Reply:
I'm not sure what the plan was for these todos. @beyelerb can you comment?

Reply:
Created https://codice.atlassian.net/browse/CAL-519 to address these TODOs